src/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx

```diff
@@ -90,6 +91,16 @@ export const SimOptionsPage = () => {
                         </div>
                     </SettingItem>
 
+                    <SettingItem name={t('Settings.SimOptions.SimBridgeIp')}>
+                        <SimpleInput
+                            className="text-center w-30"
+                            value={simbridgeIp}
+                            onChange={(event) => {
+                                setSimbridgeIp(event.replace(/\s/g, ''));
```

Comment:
It's really just an address rather than an IP address, right?

Reply:
Yep, you can have something like `simbridge.local` or if you have a proper dns setup they could have something like `computername.local` etc. Should support that :)

Reply:
What I mean is it probably shouldn't be labelled "IP Address" in the UI etc.